mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -265,7 +265,11 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_35054 setNoDrag (Z)V
 		ARG 1 noDrag
 	METHOD method_36362 updateGlowing ()V
+	METHOD method_36549 addStingerParticles ()V
```

Comment:
This is completely incorrect, the particles have nothing to do with stingers and are in fact the 'poof' particles that result from the death of the entity. In Enigma the 'stingers' field is mistakenly shown as the value being iterated over in the for loop as a result of a bug.

Reply:
Mapped correctly in #2417

Reply:
Yeah, sorry, but enigma's cfr appears to incorrectly reuse local variables. If you see something is a local variable that is not in the list of parameters, then for sure enigma's cfr broke there

Reply:
```suggestion
```